internal/feeder/rekor/rekor_feeder.go

```diff
@@ -98,7 +106,7 @@ func FeedLog(ctx context.Context, l config.Log, w feeder.Witness, c *http.Client
 			return [][]byte{}, nil
 		}
 		cp := proof{}
-		if err := getJSON(ctx, c, lURL, fmt.Sprintf("api/v1/log/proof?firstSize=%d&lastSize=%d&treeID=%s", from.Size, to.Size, l.ID), &cp); err != nil {
+		if err := getJSON(ctx, c, lURL, fmt.Sprintf("api/v1/log/proof?firstSize=%d&lastSize=%d&treeID=%s", from.Size, to.Size, treeID), &cp); err != nil {
```

Comment:
If we go with this approach, do we also need to remove the query parameter from `lURL`?

Reply:
Sorry for the delay - Awesome, that's good! This approach LGTM then.